src/components/Summary.js

```diff
@@ -71,7 +71,10 @@ export function Summary({ backendUrl }) {
   const onCopy = (e) => {
     navigator.clipboard
       .writeText(summaryRef.current.innerText)
-      .then(() => setCopied(() => true))
+      .then(() => {
+        setCopied(() => true);
+        setTimeout(() => setCopied(() => false), 5000);
```

Comment:
If I understand this correctly the message will be displayed for 5 seconds?
I think it's too long. 1 sec should be enough.
```suggestion
        setTimeout(() => setCopied(() => false), 1000);
```

Reply:
👍